src/components/screens/OpeningTimesCard.js

```diff
@@ -23,70 +26,94 @@ const DateBox = styled(TimeBox)`
 
 /* eslint-disable complexity */
 /* NOTE: we need to check a lot for presence, so this is that complex */
-export const OpeningTimesCard = ({ openingHours }) => (
-  <Wrapper>
-    {!!openingHours &&
-      openingHours.map((item, index) => {
-        const {
-          weekday,
-          timeFrom,
-          timeTo,
-          dateFrom,
-          dateTo,
-          description,
-          open,
-          useYear = false
-        } = item;
-        const returnFormatDate = useYear ? 'DD.MM.YYYY' : 'DD.MM.';
-
-        return (
-          <View key={index} style={index !== openingHours.length - 1 ? styles.divider : null}>
-            {!!weekday && <BoldText style={styles.marginBottom}>{weekday}</BoldText>}
-
-            {(!!timeFrom || !!timeTo || !!dateFrom || !!dateTo) && (
-              <WrapperRow>
-                {(open === undefined || open === true) && (!!timeFrom || !!timeTo) && (
-                  <TimeBox>
-                    {!!timeFrom && <RegularText>{timeFrom}</RegularText>}
-                    {!!timeFrom && !!timeTo && <RegularText> -</RegularText>}
-                    {!!timeTo && <RegularText> {timeTo}</RegularText>}
-                  </TimeBox>
-                )}
-                {open === false && (
-                  <TimeBox>
-                    <RegularText>geschlossen</RegularText>
-                  </TimeBox>
-                )}
-                {(!!dateFrom || !!dateTo) && (
-                  <DateBox>
-                    {!!dateFrom && (
-                      <RegularText>
-                        <RegularText small />
-                        {momentFormat(dateFrom, returnFormatDate)}
-                      </RegularText>
-                    )}
-
-                    {!!dateTo && dateTo !== dateFrom && (
-                      <RegularText>
-                        <RegularText small>bis </RegularText>
-                        {momentFormat(dateTo, returnFormatDate)}
-                      </RegularText>
-                    )}
-                  </DateBox>
-                )}
-              </WrapperRow>
-            )}
-
-            {!!description && (
-              <WrapperRow>
-                <HtmlView html={description} />
-              </WrapperRow>
-            )}
-          </View>
-        );
-      })}
-  </Wrapper>
-);
+export const OpeningTimesCard = ({ openingHours }) => {
+  const [moreData, setMoreData] = useState(1);
+
+  const loadMoreItems = () => {
+    setMoreData((prev) => prev + 1);
+  };
+
+  const filteredOpeningHoursFromToday =
+    openingHours?.filter((item) => {
```

Comment:
i have changed to `openingHours?.filter` and added a `|| []` fallback, to be able to always do array operations on `filteredOpeningHoursFromToday`